app/controllers/api/v1/cars_controller.rb

```diff
@@ -0,0 +1,42 @@
+class Api::V1::CarsController < ApplicationController
+  def index
+    @cars = Car.all
```

Comment:
- [ ] You don't really need to use instance variable here, since you are only using `@cars` here in the controller and only in this method.